spec/gcov/gcov_test_cases_spec.rb

```diff
@@ -194,4 +194,147 @@ def can_create_html_report
       end
     end
   end
+
+  def can_create_gcov_html_report_from_crashing_test_runner_with_enabled_debug_and_cmd_args_set_to_true_for_test_cases_not_causing_crash
+    @c.with_context do
+      Dir.chdir @proj_name do
+        FileUtils.cp test_asset_path("example_file.h"), 'src/'
+        FileUtils.cp test_asset_path("example_file.c"), 'src/'
+        FileUtils.cp test_asset_path("test_example_file_sigsegv.c"), 'test/'
+        FileUtils.cp test_asset_path("project_with_guts_gcov.yml"), 'project.yml'
+
+        add_line = false
+        updated_prj_yml = []
+        File.read('project.yml').split("\n").each do |line|
```

Comment:
Hi @mvandervoord ,

May I suggest to change this part responsible by updating project.yml to look:

```
File.read('project.yml').split("\n").each do |line|
          if line =~ /\:use_backtrace_gdb_reporter\:/
            line = line.gsub("FALSE", "TRUE")
          end
          updated_prj_yml.append(line)
        end
```

Reply:
Thank your for the advice! I'm rolling this into a helper. :)